openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -65,22 +68,22 @@ <h2 class="staff-info__title ui-staff__title">
                     </div>
 
                     <div class="staff-info__status ui-staff__content__section">
-                        <a aria-role="button" href="" id="schedule_training" class="action--submit"><span class="copy">{% trans "Schedule Example-Based Assessment Training" %}</span></a>
-                        <div id="schedule_training_message"></div>
+                        <button class="action--submit action--submit-training"><span class="copy">{% trans "Schedule Example-Based Assessment Training" %}</span></button>
```

Comment:
I was at least able to press these buttons and verify that the appropriate confirmation messages show.